src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs

```diff
@@ -765,7 +766,9 @@ protected Size AddViewAtOffset(View child, GeneratorDirection direction, int ext
 
 			size = ApplyChildStretch(size, slotSize, viewType);
 
+			UnoViewGroup.StartLayoutingFromMeasure();
```

Comment:
You can check if child.IsInLayout is true (which will be the case sometimes, this may get called either during measure or during layout). When it's true, the Start/End.. isn't necessary.

Reply:
Will do